### DISCUSSION TITLE
Unicode fonts (add_font) with Templates?

### DISCUSSION THREAD

**Discussion by professorlance** [py-pdf/fpdf2#1239] [Q&A] [answered]
Hello all,
Is there any way to use custom unicode fonts with a template? When I try to use the add_font() method with a template, I receive an attribute error.
Thanks!

**Comment by gmischler** [comment, marked as answer]
Welcome to fpdf2, @professorlance !

The templates are entirely agnostic of font types. They will use whatever fonts have been added **_to the document_** beforehand.
As you will find confirmed in the [documentation](https://py-pdf.github.io/fpdf2/Templates.html), the template instances themselves don't have an `add_font()` method. If you tried that, then an attribute error is inevitable.

**Reply by professorlance** [reply]
Hi Georg, thank you for the feedback!
